package.json

```diff
@@ -21,8 +21,11 @@
     "@docusaurus/theme-mermaid": "3.0.1",
     "@easyops-cn/docusaurus-search-local": "^0.37.4",
     "@hyperlane-xyz/core": "3.1.10",
+    "@hyperlane-xyz/sdk": "3.9.0",
```

Comment:
These two should match. In fact we don't need the core reference because the sdk depends on it anyway